Automattic/jetpack
# CRM: Updating core Noto Sans fonts and adding new CJK fonts

Fixes https://github.com/Automattic/zero-bs-crm/issues/1782

## Proposed changes:

* This PR updates the `$font_json` variable with the five new CJK fonts: 
  * NotoSansJP (now NotoSansJapanese) -> Japanese
  * NotoSansKR (now NotoSansKorean) -> Korean
  * NotoSansSC (now NotoSansSimplifiedChinese) -> Simplified Chinese
  * NotoSansTC (now NotoSansTaiwan)-> Traditional Chinese - Taiwan
  * NotoSansHK (now NotoSansHongKong) -> Traditional Chinese - Hong Kong
* It also updates the `pdffonts.zip` file with new core Noto Sans font versions, which apart from updating several-year-old files also means that the two new currencies added in this PR will be viewable on PDF as well: https://github.com/Automattic/jetpack/pull/36897

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/1782

p1713366111446779-slack-CTXBP902X

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:


* To test, check out this PR or create a JN site using this PR via the beta tester plugin.
* Via the locale settings page - `wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=locale` - set your currency to be the Azerbaijan Manat ('₼', 'AZN'), or the  Kazakhstani Tenge ('₸', 'KZT').
* Then, at the bottom of the page re-install the default fonts using the 'refresh' icon.
* On an invoice, select to download the PDF. The new currency should now be visible.
* In trunk, you won't see the currency on the PDF, only a blank square.
* To test CJK, you can test if you are able to add Korean, Chinese, or Japanese characters to (for example) an invoice description, after installing the relevant font.
* To see the font via PDF, modify the template with the new font added to the CSS styles, as [per this link](https://kb.jetpackcrm.com/knowledge-base/using-custom-fonts-in-crm-pdfs-e-g-invoice-templates/).
